FLAIR.md

```diff
@@ -0,0 +1,29 @@
+Acknowledgments
+=========
+
+<p valign="middle"><a href="https://www.browserstack.com/"><img src="http://media.archiveofourown.org/ao3/logos/Browserstack.svg" width="200"/></a> for cross-browser testing tool. </p>
```

Comment:
You put a period at the end of the Browserstack description, but for most of the other tools we use there isn't a period. Just need to be consistent, whichever way we go (I lean towards periods).